inst/pages/04_containers.qmd

```diff
@@ -736,7 +736,7 @@ GlobalPatterns_phyloseq2
 
 Conversion is possible between other data formats. Interested readers can refer to the following functions:
 
-* [makeTreeSummarizedExperimentFromDADA2](https://microbiome.github.io/mia/reference/makeTreeSummarizedExperimentFromDADA2.html)  
-* [makeSummarizedExperimentFromBiom](https://microbiome.github.io/mia/reference/makeSummarizedExperimentFromBiom.html)  
+* [makeTreeSummarizedExperimentFromDADA2](https://microbiome.github.io/mia/reference/makeTreeSEFromDADA2.html)  
+* [makeSummarizedExperimentFromBiom](https://microbiome.github.io/mia/reference/makeTreeSEFromBiom.html)  
```

Comment:
Is this correct? The link should point to a function that has the same name than the link. Here the link name and URLs are different?

Reply:
Here the link names and URLs are still not identical 

makeTreeSummarizedExperimentFromDADA2
makeTreeSEFromDADA2.html

makeSummarizedExperimentFromBiom
makeTreeSEFromBiom.html

Reply:
I have changed it. Please refer to the new commit.